.github/ci/benchmark/tigerbeetle.hcl

```diff
@@ -99,7 +99,7 @@ git clone ${var.git_url}
 cd tigerbeetle
 git checkout ${var.git_ref}
 ./scripts/install_zig.sh
-./zig/zig build install -Drelease -Dconfig=production
+./zig/zig build -Drelease -Dconfig=production
```

Comment:
We can remove this whole folder too, unless you think it's useful to point people to... It's not actively used.

Reply:
:+1: